api/types/types.go

```diff
@@ -102,9 +102,10 @@ type ContainerStats struct {
 // Ping contains response of Engine API:
 // GET "/_ping"
 type Ping struct {
-	APIVersion   string
-	OSType       string
-	Experimental bool
+	APIVersion      string
+	OSType          string
+	Experimental    bool
+	FeatureBuildKit *bool
```

Comment:
If we want *more* feature flag in the future, maybe we should make this one a struct or a map 👼 

```
Features map[string]bool
```

Reply:
*cough* `map[string]string` *cough*

Reply:
@cpuguy83 I think `map[string]bool` is actually better here, which will satisfy the tri-state we discussed earlier.

Reply:
The reason I say string is because you can change strings and represent quite a bit more.... but sounds like people think it may be overkill. That's fine.